### ISSUE TITLE
MD031 not ignoring comments

### ISSUE THREAD

**Issue by acarnefixw** [rvben/rumdl#866]
### rumdl version

0.2.60

### Installation method

Other

### Bug description

When I have a code block with a comment above it, the code block gets flagged by rule MD031.
For example:

````markdown
Sample text

<!-- comment -->
```python
print("Hello")
```
````

It looks like `markdownlint` had a similar [issue](https://github.com/DavidAnson/markdownlint/issues/171) in the past.
My specific use case is using comments to trigger a custom `mkdocs` plugin that uses the code block content, so it would be ideal if I could leave the comments right above the code block.

### Steps to reproduce

1. Create a file `test.md` with this content:

    ````markdown
    # Header
    
    <!-- comment -->
    ```python
    print("Hello")
    ```
    ````

1. Run `rumdl check test.md`
1. See error: `test.md:4:1: [MD031] No blank line before fenced code block [*]`

### Expected behavior

No error since the comment should be considered a blank line

### Actual behavior

Error is reported: `test.md:4:1: [MD031] No blank line before fenced code block [*]`

### Configuration

```toml

```

### Operating system

Linux

**Comment by rvben** [comment]
Thanks for reporting this, and for explaining the MkDocs use case. MD031 now treats a comment-only line as a blank line. The fix is on `main` in `b394e303`.
